src/main/java/de/tum/in/www1/artemis/web/rest/QuizExerciseResource.java

```diff
@@ -428,6 +424,7 @@ public ResponseEntity<QuizExercise> performActionForQuizExercise(@PathVariable L
                 // set release date to now, truncated to seconds because the database only stores seconds
                 var now = ZonedDateTime.now().truncatedTo(ChronoUnit.SECONDS);
                 quizBatchService.getOrCreateSynchronizedQuizBatch(quizExercise).setStartTime(now);
+                quizExercise.setReleaseDate(now);
```

Comment:
This should probably only happen when the old release date is in the future and preserve the old release date when the exercise was already released.
```suggestion
                if (quizExercise.getReleaseDate() != null && quizExercise.getReleaseDate().isAfter(ZonedDateTime.now()) {
                    quizExercise.setReleaseDate(now);
                }
```